### PR TITLE
Feature/AB#15346 Org Info & AB#23421 Signing Authority

### PR DESCRIPTION
**Pull request by don-aot**
[AB#15346](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/15346):Organization information accessible in the same place and editable,
[AB#23421](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23421):Signing Authority Contact Info

### CODE REVIEW

**Comment by AndreGAot** [comment]
@don-aot  - Hi Don, have confirmed that the signing authority belongs to the application

**Comment by don-aot** [comment]
> @don-aot - Hi Don, have confirmed that the signing authority belongs to the application

Updated the code and moved Signing Authority to the Application Table.

File: applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs [hunk 1]
```diff
@@ -27,4 +27,21 @@ public static class ProjectInfoOptionsList
             new KeyValuePair<string, string>("NO", "No"),
       });
 
+     public static ImmutableDictionary<string, string> OrganizationTypeList =>
+     ImmutableDictionary.CreateRange(new[]
+       {
+              new KeyValuePair<string, string>("CORPORATION", "Corporation"),
+              new KeyValuePair<string, string>("PARTNERSHIP", "Partnership"),
+              new KeyValuePair<string, string>("INCORPORATED_COOPERATIVE", "Incorporated Cooperative"),
+              new KeyValuePair<string, string>("FIRSR_NATION", "First Nation or Indigenous-Owned Enterprises"),
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs:36]: spelling FIRST_NATION ?

**Comment by AndreGAot** [comment]
@don-aot - please can we add some unit tests, have left 1 other comment, other testing good so far

File: applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs (hunk 1 above)
Comment (don-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs]: > spelling FIRST_NATION ?

@AndreGAot  Corrected the spelling mistake.

**Comment by don-aot** [comment]
> @don-aot - please can we add some unit tests, have left 1 other comment, other testing good so far

Pushed the unit testing for the new component

**Comment by AndreGAot** [comment]
@don-aot - please can you check the Sector / Subsector fields, they are not behaving correctly, they look correct at the API level so think it is in the page / viewmodel and JS

**Comment by don-aot** [comment]
> @don-aot - please can you check the Sector / Subsector fields, they are not behaving correctly, they look correct at the API level so think it is in the page / viewmodel and JS

@AndreGAot  FIxed the error with dropdown

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**